api/serverless.yml

```diff
@@ -26,13 +26,18 @@ functions:
           path: /infected
           method: post
           integration: lambda-proxy
-  getDevicesNearInfected:
-    handler: handler.getDevicesNearInfected
+  getDeviceRisk:
+    handler: handler.getDeviceRisk
     events:
       - http:
-          path: /devices-near-infected
+          path: /device-risk
```

Comment:
I think you need to edit the Swagger template, and the API Gateway compatible (Removing the example fields) so this could deploy correctly